mantidproject/mantid
# Add help button to add function dialog in fitting

~~**Do not merge until #27908 has been merged**~~

**Description of work.**
This PR adds a help button into the add function dialog in fitting. 
It also fixes a small bug where the fitting docs were never found as the url had changed from `fitfunctions/function_name` to `fitting/fitfunctions/function_name`

Report to: @AnthonyLim23 

**To test:**
Build the fitting docs
Open workbench, load a workspace and do a plot.
Click the fit button
Right-click on the plot and click any of the `add ...` options.
There should be a `?` button in the bottom left corner.
If you click it then it should show the help page for the fit function selected. If the function does not have documentation  then it should display the fit functions index page. 

Similarly Right-click on the property browser and click add function.
There should be a `?` in the bottom left corner which works the same as above

Add a function and right click on it in the function browser.
Click help and it should load the docs for that function if it exists

Fixes #27020 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
